snowplow-tracker/src/androidTest/java/com/snowplowanalytics/snowplow/tracker/RequestResultTest.java

```diff
@@ -0,0 +1,54 @@
+package com.snowplowanalytics.snowplow.tracker;
+
+import android.test.AndroidTestCase;
+
+import com.snowplowanalytics.snowplow.network.RequestResult;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public class RequestResultTest extends AndroidTestCase {
```

Comment:
Same comment of the other PR - AndroidTestCase is deprecated.
We should migrate all the tests to the new way, soon or later.

Reply:
Changed!